src/components/StatCharts/StatCharts.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import BigNumber from "bignumber.js";
+import { hashesToGH } from "../formatters";
+import { hexToNumber } from "@etclabscore/eserialize";
+import { Grid } from "@material-ui/core";
+import ChartCard from "../ChartCard";
+import { VictoryLine, VictoryBar, VictoryChart } from "victory";
+import { useTranslation } from "react-i18next";
+
+const config = {
+  blockTime: 15, // seconds
+  blockHistoryLength: 100,
+  chartHeight: 200,
+  chartWidth: 400,
+};
+
+const blockMapGasUsed = (block: any) => {
+  return {
+    x: hexToNumber(block.number),
+    y: new BigNumber(block.gasUsed).dividedBy(1000000),
+  };
+};
+
+const blockMapUncles = (block: any) => {
+  return {
+    x: hexToNumber(block.number),
+    y: block.uncles.length,
+  };
+};
+
+const blockMapHashRate = (block: any) => {
+  return {
+    x: hexToNumber(block.number),
+    y: hashesToGH(new BigNumber(block.difficulty, 16).dividedBy(config.blockTime)),
```

Comment:
Mmmm.. Using a hardcoded block time is a liiiitleee stretchy?

Reply:
yea would be nice to calculate the block time for the list of blocks you are looking at. maybe a helper `getBlockTimeFromBlocks(blocks)`